libbeat/docs/loggingconfig.asciidoc

```diff
@@ -206,6 +206,13 @@ reported by the local system clock. All other intervals are calculated from the
 unix epoch. Defaults to disabled.
 endif::serverless[]
 
+[bool]
```

Comment:
This needs to be `[float]`. It is a formatting instruction for Asciidoc (https://asciidoctor.org/docs/user-manual/#discrete-headings).